src/training/_hooks.py

```diff
@@ -119,7 +123,7 @@ def after_run(self, context, values):
       throughput = self._global_steps_completed * float(self._batch_size) / float(duration)
 
       logging.info(LogTrainingHook._MESSAGE_FORMAT,
-                   self._global_steps_completed, duration, throughput, loss_value)
+                   self._global_steps_completed, loss_value)
```

Comment:
General comment on the log change based on the description of the CL ... this looks like logging the same thing multiple times, which I think you are addressing, but it isn't once you factor distributed training. Each worker might have different throughput values.

Maybe we need to refactor differently -- a worker logger and a master logger -- but I haven't fully thought through that. In fact, we might be able to achieve this now with just using LogTrainingHook and not LogSessionHook in the case of a single node training job.